web/app.js

```diff
@@ -2759,6 +2780,18 @@ function onKeyUp(evt) {
 function onKeyDown(evt) {
   this._isCtrlKeyDown = evt.key === "Control";
 
+  if (
+    this.editorUndoBar?.isOpen &&
+    evt.keyCode !== 9 &&
+    evt.keyCode !== 16 &&
+    (!(evt.keyCode === 13 || evt.keyCode === 32) ||
+      getActiveOrFocusedElement() !== this.appConfig.editorUndoBar.undoButton)
```

Comment:
Nit: This could be "simplified" a little bit by moving the negation outside.
```suggestion
    !((evt.keyCode === 13 || evt.keyCode === 32) &&
      getActiveOrFocusedElement() === this.appConfig.editorUndoBar.undoButton)
```

Reply:
okay, done!